en/exercises/web_vulnerability_assessment/manual_testing_with_burp.md

```diff
@@ -0,0 +1,48 @@
+**Introduction**
+
+According to Burp's official [documentation](https://support.portswigger.net/customer/portal/articles/1783101-how-to-use-burp-suite), Burp Suite is an integrated platform for performing security testing of web applications. It is **not a point-and-click** tool, but is designed to be used by hands-on testers to support the testing process. With a little bit of effort, anyone can start using the core features of Burp to test the security of their applications. Some of Burp's more advanced features will take further learning and experience to master. All of this investment is hugely worth it - Burp's user-driven workflow is by the far the most effective way to perform web security testing, and will take you way beyond the capabilities of any conventional point-and-click scanner. Burp is intuitive and user-friendly, and the best way to start learning is by doing. These steps will get you started with running Burp and using its basic features. You can then read on deeper into the documentation to become more proficient in using this supremely powerful tool. 

```

Comment:
Adding reference to the original documentation

Reply:
It looks like a lot of the burp walkthrough is from their own documentation - in cases where the tool actually is well documented (and appears to keep that documentation more or less up to date with the tool), it's best to just point auditors there for the specific tool-level needs, and instead provide a more "meta" guide -- see the exercises covering nmap, wireshark, and recon-ng for some variations adapting to different levels of documentation from the tool providers.

Reply:
Got it @joncamfield!